utils/src/check/super.c

```diff
@@ -65,15 +68,53 @@ static int check_super_flag(__le64 flags)
 	return 0;
 }
 
-static int check_quorum_config(struct scoutfs_quorum_config qconf)
+/*
+ * After checking the Quorum config we check the Quorum slots of the meta device
+ * for addressing issues
+ */
+static int check_quorum_slots(struct scoutfs_quorum_slot *slots)
 {
-	uint64_t qversion = le64_to_cpu(qconf.version);
+	uint16_t family;
+	int port;
+	struct in_addr in;
+
+	for (int i = 0; i < SCOUTFS_QUORUM_MAX_SLOTS; i++) {
+		port = le16_to_cpu(slots[i].addr.v4.port);
+		debug("Quorum slot %u port: %d\n", i, port);
+		if ((port <= 0) || (port > USHRT_MAX))
+			problem(PB_QSLOT_BAD_PORT, "Quorum Slot %u has bad port", i);
+
+		family = le16_to_cpu(slots[i].addr.v4.family);
+		debug("Quorum slot %u family: %u\n", i, family);
+		if (family != SCOUTFS_AF_IPV4)
```

Comment:
Take a look at `util/quorum.c` at `bool valid_quorum_slots` and you can see how the code uses the `family` bit to determine whether a quorum slot is used or not using `SCOUTFS_AF_NONE`. We need to do the same here.